bin/dart.bat

```diff
@@ -0,0 +1,46 @@
+@ECHO off
+REM Copyright 2014 The Flutter Authors. All rights reserved.
+REM Use of this source code is governed by a BSD-style license that can be
+REM found in the LICENSE file.
+
+REM ---------------------------------- NOTE ----------------------------------
+REM
+REM Please keep the logic in this file consistent with the logic in the
+REM `flutter` script in the same directory to ensure that Flutter continues to
```

Comment:
Does this comment need updating (i.e. is the linux side also getting restructured?)

Reply:
Ok, that's fine

Reply:
Done